dvc/command/completion.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import io
+import logging
+import sys
+
+from dvc.command.base import CmdBase, append_doc_link
+
+logger = logging.getLogger(__name__)
+GLOBAL_OPTIONS = ["-h", "--help", "-q", "--quiet", "-v", "--verbose"]
+ROOT_PREFIX = "_dvc"
+UNCOMPLETABLE_POSITIONALS = {
+    "rev",
+    "url",
+    "args",
+    "name",
+    "option",
+    "value",
+    "command",
```

Comment:
This is hacky, but might be okay as a first step. Ideally we need some explicit way of telling this completion lib what to complete and with what.

Reply:
outdated - removed